pontoon/base/models.py

```diff
@@ -1418,6 +1418,16 @@ def check_latest_translation(self, instance):
             instance.latest_translation = self
             instance.save(update_fields=['latest_translation'])
 
+    def discard(self, stats=True):
```

Comment:
Let's rename `discard` to `unapprove` everywhere in the code.

We'll replace delete with discard sometime in the future:
https://bugzilla.mozilla.org/show_bug.cgi?id=1192872